tools/end2end_tests/base/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM debian:latest
```

Comment:
Don't use `:latest` as this will ding our OpenSSF Scorecard. Instead should be pinned to a SHA, and a Dependabot job set up to keep it updated.

Whenever possible we should use `slim` rather than full distros e.g. `debian:stable-20230612-slim@sha256:b09f68bffcf9c14f3105f262e92321d05abaf48460d1f43f884325bcd4395b95` is presently what we use for the sshnpd [Dockerfile](https://github.com/atsign-foundation/sshnoports/blob/trunk/Dockerfile)

Reply:
The easiest way is to look at other Dockerfiles in our repos. But more generally:

```
docker buildx imagetools inspect $IMAGE --format "{{json .Manifest}}" | jq -r .digest
```

e.g.

```
docker buildx imagetools inspect debian:stable-20230612-slim --format "{{json .Manifest}}" | jq -r .digest
```

will give you:

```
sha256:b09f68bffcf9c14f3105f262e92321d05abaf48460d1f43f884325bcd4395b95
```

Reply:
https://github.com/atsign-foundation/sshnoports/pull/212/commits/bd964acdcd1a038af73ac6c89df250a2086aa6ff